### ISSUE TITLE
Best result using only diffs

### ISSUE THREAD

**Issue by gewoonrik** [gousiosg/pullreqs-dnn#3]
Epoch 4/30
48482/48482 [==============================] - 330s - loss: 0.6224 - acc: 0.6464 - fmeasure: 0.6516 - val_loss: 0.6666 - val_acc: 0.6316 - val_fmeasure: 0.6075

./train.py --prefix=ruby_code_heuristic --batch_size=256 --epochs=30 --dropout=0.3 --lstm_output=400 --embedding_output=700

using learningrate: 0.005 and optimizer: RMSprop
